@ndlib/gatsby-theme-marble/src/components/Internal/DropDown/test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { act } from 'react-dom/test-utils'
+import DropDown from './'
+
+describe('DropDown', () => {
+  const mockSetState = jest.fn()
+  jest.mock('react', () => ({
+    useState: initial => [initial, mockSetState],
+  }))
+  test('render and click', () => {
+    let wrapper
+    const sxStyle = {
+      wrapper: {},
+      toggle: {},
+      optionsOpen: { display: 'block' },
+      optionsClosed: { display: 'none' },
+    }
+    const buttonLabel = <div className='button' />
+    const options = <div className='options' />
+
+    act(() => {
+      wrapper = shallow(<DropDown sxStyle={sxStyle} buttonLabel={buttonLabel} options={options} />)
+    })
```

Comment:
Did not know about this. Might be useful. Out of curiosity, have you noticed any significant differences doing this vs. simply defining wrapper as a constant without an act function? Docs say it `makes your test run closer to how React works in the browser` but I'm not really sure what that means in practice. 😄

Reply:
I haven't noticed any difference in testing speed, but it was basically the only way I could reliably get the `useState` stuff to fire correctly.